docs/docs/admin/alerts/create.md

```diff
@@ -3,16 +3,12 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<!-- prettier-ignore -->
-!!! Warning "Usage restricted"
-    You need to be administrator to use this feature.
-
 Alerts can be used to inform users about changes, news or maintenance work. The
 alerts are displayed to each user.
 
 <!-- prettier-ignore-start -->
 
-1. Please navigate to `Profile` → `Settings`
+1. Please navigate to `Menu` → `Settings`
```

Comment:
This isn't necessarily a big deal and I know it already exists in the docs, but using "Please" in these contexts is unusual (since this is a set of instructions and not a request)

Reply:
Oh yes, someone mentioned this some time ago already. I keep doing the same mistakes 🤦‍♂️

Reply:
Thanks for mentioning it!